packages/spacecat-shared-utils/src/bot-blocker-detect/bot-blocker-detect.js

```diff
@@ -32,22 +33,101 @@ const DEFAULT_TIMEOUT = 5000;
 function analyzeResponse(response) {
   const { status, headers } = response;
 
-  if (status === 403 && headers.get('cf-ray')) {
+  // Check for CDN/blocker infrastructure presence
+  const hasCloudflare = headers.get('cf-ray') || headers.get('server') === 'cloudflare';
+  const hasImperva = headers.get('x-iinfo') || headers.get('x-cdn') === 'Incapsula';
+  const hasAkamai = headers.get('x-akamai-request-id')
+    || headers.get('x-akamai-session-id')
+    || headers.get('server')?.includes('AkamaiGHost');
+  const hasFastly = headers.get('x-served-by')?.includes('cache-')
```

Comment:
`includes('cache-')` - it can give a false positive results, because other CDN can have the same `cache-` prefix

Reply:
changed to startsWith